Personal_Accounts/src/test/java/org/openapitools/api_tests/aisp_accounts_accountId_get.feature

```diff
@@ -16,18 +16,18 @@ Background:
   * def authToken = karate.properties['AUTH_TOKEN']
 
 Scenario Outline: Validate response for valid account id
-  Given path '/aisp/accounts/', '<accountId>'
+  Given path '/aisp/accounts/', <accountId>
   And headers {Authorization: '#(authToken)', 'x-fapi-auth-date': 'Sun, 10 Sep 2017 19:43:31 UTC', 'x-fapi-customer-ip-address': '169.254.169.254', 'x-fapi-interaction-id': 'uniqueId', 'Accept-Language': 'en-US'}
   When method get
   Then status 200
-  And match responseHeaders['Content-Type'] == 'application/json'
-  And match response.data.account[*].accountId == '<accountId>'
-  And match response.data.account[*].accountNumber == '#string'
-  And match response.data.account[*].accountType == '#string'
-  And match response.data.account[*].accountSubType == '#string'
-  And match response.data.account[*].productName == '#string'
-  And match response.data.account[*].accountStatus == '#string'
-  And match response.data.account[*].currency == '#regex ^[A-Z]{3,3}$'
+  And match responseHeaders['Content-Type'] == ['application/json']
+  And match response.data.account[0].accountId == '#string'
```

Comment:
Instead of `*` `0`th  index should be accessed. And request account ID will not necessarily be same as the response one